Model/GridExport/Type/Xml.php

```diff
@@ -18,11 +19,11 @@ class Xml extends AbstractTypeType
 
     private ExcelFactory $excelFactory;
 
-    private SourceIteratorFactory $sourceIteratorFactory;
+    private IteratorFactory $sourceIteratorFactory;
```

Comment:
`SourceIteratorFactory` ist besser, oder noch besser `ExportSourceIteratorFactory`
Wenn eine Klasse importiert wird sollte der Name noch beschreiben um was es geht.
Die alte Konvention von Magento 1 das jeder Namespace und die Klasse einem Wort entspricht funktioniert mit PHP 5.6 Namespaces nicht mehr gut. Der Code wird weniger verständlich durch die Imports, und man braucht für jede Klasse beim Import ein Alias.
Besser ist es ein paar Wörter aus dem namespace in den Namen der Klasse zu übernehmen, dann ist ein Aliasing nicht mehr nötig.